From e419d12dee049e3b6c5bec6c85cb8f790cc6e1de Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Fri, 1 Aug 2025 07:33:31 +0200
Subject: [PATCH 13/55] base-files: force users to log in on the serial console

This causes the serial console to require a password when logging in.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/base-files/files/usr/libexec/login.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/package/base-files/files/usr/libexec/login.sh b/package/base-files/files/usr/libexec/login.sh
index e2f898e850..32bb997f6e 100755
--- a/package/base-files/files/usr/libexec/login.sh
+++ b/package/base-files/files/usr/libexec/login.sh
@@ -12,6 +12,4 @@
 	esac
 }
 
-[ "$(uci -q get system.@system[0].ttylogin)" = 1 ] || exec /bin/login -f root
-
 exec /bin/login
-- 
2.34.1

